src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -126,28 +121,20 @@ public void startsDaemon() throws Exception {
      */
     @Test
     public void deprecatesDefaultImage() throws IOException {
+        final String rultor = "yegor256/rultor";
         final Talk talk = this.talk();
         final XML xml = talk.read();
+        final String dir = xml.xpath("/talk/daemon/dir/text()").get(0);
         final List<String> repos = xml.xpath("/wire/github-repo/text()");
-        if (
-            repos.isEmpty() || !StartsDaemonITCase.RULTOR.equals(repos.get(0))
+        if ((repos.isEmpty() || !rultor.equals(repos.get(0)))
+            && xml.xpath(
+                "/p/entry[@key='merge']/entry[@key='script']"
+            ).contains(rultor)
         ) {
-            for (
-                final String path : xml.xpath(
-                    "/p/entry[@key='merge']/entry[@key='script']"
-                )
-            ) {
-                if (StartsDaemonITCase.RULTOR.equals(path)) {
-                    final String dir = talk.read()
-                        .xpath("/talk/daemon/dir/text()").get(0);
-                    MatcherAssert.assertThat(
-                        dir,
-                        StringStartsWith.startsWith(
-                            "#### Deprecation Notice ####"
-                        )
-                    );
-                }
-            }
+            MatcherAssert.assertThat(
+                dir,
+                StringStartsWith.startsWith("#### Deprecation Notice ####")
+            );
```

Comment:
@xupyprmv this doesn't really change anything the matcher is still wrapped in an if :) If the if evaluates to `false` this test passes without testing anything. The matching needs to be outside the if.